HotelManagerment/com.spring.sample/src/com/spring/sample/entity/Receipt.java

```diff
@@ -0,0 +1,64 @@
+package com.spring.sample.entity;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class Receipt extends BaseEntity implements Serializable {
+
+    private String rentalReceiptID; // Updated from Long to String
```

Comment:
```suggestion
    private String id; // Updated from Long to String
```
cơ mà khóa chính tiên quyết nên là Long nha, vì sao lại update sang string thế

Reply:
Dạ tại ban đầu nó bị bug, em chuyển thành String lại chạy được ngay ạ.

Reply:
thế thì em coi lại trường trogn DB của em đang là kiểu j nhá, chứ id là int mới đúng nha, chứ string là ko đúng nà